elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -146,19 +167,6 @@ defmodule API.Client.Channel do
     {:noreply, socket}
   end
 
-  # This event is sent when client or group are changed (eg. renamed, verified, etc.),
-  # so we just re-initialize the client the same way as after join to push the updates
-  def handle_info(:updated, socket) do
-    OpenTelemetry.Ctx.attach(socket.assigns.opentelemetry_ctx)
-    OpenTelemetry.Tracer.set_current_span(socket.assigns.opentelemetry_span_ctx)
-
-    OpenTelemetry.Tracer.with_span "client.updated" do
-      socket = assign(socket, client: Clients.fetch_client_by_id!(socket.assigns.client.id))
-      :ok = init(socket)
-      {:noreply, socket}
-    end
-  end
```

Comment:
Nit: Re-ordering these bloats the diff a fair bit which makes it hard to see what actually changed.